app/controllers/tasks.js

```diff
@@ -33,16 +38,34 @@ export default class TasksController extends Controller {
   @tracked message = ''; // this is required in the modal
   @tracked buttonRequired = false; // this is required in the modal
   @tracked disabled = false; // this is required for the holder component
+  @tracked inProgressTasks = this.model.filter(
+    (task) => task.status === 'IN_PROGRESS'
+  );
+  // @tracked noInProgressTask = this.inProgressTasks.length === 0;
+  @tracked findingTask = false;
+  @tracked showFetchButton = this.inProgressTasks.length === 0;
 
   @action toggleDropDown() {
     this.showDropDown = !this.showDropDown;
   }
 
+  tasksInProgress() {
+    return this.model.filter((task) => task.status === 'IN_PROGRESS');
+  }
+
+  noInProgressTask() {
+    const taskINProgress = this.model.filter(
+      (task) => task.status === 'IN_PROGRESS'
+    );
+    this.showFetchButton = taskINProgress.length === 0;
```

Comment:
```suggestion
    this.showFetchButton = this.taskINProgress().length === 0;
```

Reply:
this.taskINProgress is not a function why do we need to call it?